device_doctor/lib/src/ios_device.dart

```diff
@@ -111,6 +115,26 @@ class IosDeviceDiscovery implements DeviceDiscovery {
     return healthCheckResult;
   }
 
+  @visibleForTesting
+  Future<HealthCheckResult> batteryLevelCheck({ProcessManager processManager}) async {
+    HealthCheckResult healthCheckResult;
+    try {
+      final String batteryCheckResult = await eval(
+          'ideviceinfo', <String>['-q', 'com.apple.mobile.battery', '-k', 'BatteryCurrentCapacity'],
+          processManager: processManager);
+      final int level = int.parse(batteryCheckResult.isEmpty ? '0' : batteryCheckResult);
+      if (level < _kBatteryMinLevel) {
+        healthCheckResult =
+            HealthCheckResult.failure(kBatteryLevelCheckKey, 'Battery level ($level) is below $_kBatteryMinLevel');
+      } else {
+        healthCheckResult = HealthCheckResult.success(kBatteryLevelCheckKey);
+      }
+    } on BuildFailedError catch (error) {
+      healthCheckResult = HealthCheckResult.failure(kKeychainUnlockCheckKey, error.toString());
```

Comment:
Is kKeychainUnlockCheckKey the right value here?

Reply:
Good catch! updated.